reddit.js

```diff
@@ -1,5 +1,73 @@
 var reddit = require("./data.json")
-
 // Write your solutions below
 
-console.log(reddit["data"]);
\ No newline at end of file
+// console.log(reddit["data"]);
+
+
+//---------------------------------------------------------
+//Question 1
+//console.log all titles
+
+reddit["data"].children.forEach(function(child) {
+	console.log(child.data.title);
+});
+
+
+//---------------------------------------------------------
+//Question 2
+//Return an array of permalinks, append http://reddit.com to each so that they contain full http:// path to reddit
+
+var getLinks = reddit["data"].children.map(function(ind) {
```

Comment:
Here, I think ```child``` might be a more suitable parameter name than ```ind```, for readability purposes and to prevent confusion.